common/src/main/java/com/alibaba/fescar/common/util/StringUtils.java

```diff
@@ -43,6 +43,44 @@ public static final boolean isEmpty(String str) {
         return (str == null) || (str.isEmpty());
     }
 
+    /**
+     * Is blank string ?
+     * @param str
+     * @return
+     */
+    public static boolean isBlank(String str) {
```

Comment:
How about use the `apache commons.lang.jar/lang3.jar` ?   The common jar also have the ` StringUtils.isBlank` method

Reply:
apache commons   also have `CollectionUtils.isBlank` method

Reply:
@lovepoem I think it's better to dependent as little as possible on the third-party packages, if the apache jar is not used, we can exclude it.